code/src/components/app-link/AppLink.scss

```diff
@@ -0,0 +1,19 @@
+@import "@design-system";
+
+.spa-link {
+  text-decoration: none;
+  color: var(--spa-link-primary-color, $black);
+  transition: color .3s;
```

Comment:
I think it's better to add a custom property for the `transition` also with duration like `mixin` to make it even more configurable. 
The `mixin` will accept two arguments like `$property` and `$duration`
and use it like
```jsx
transition: $property $duration
```
Can create `mixin` - it's reusable approach with our `design system` and not hardcoded value in this case

Reply:
I think creating this mixin does not make much sense as it just has same logic inside

Reply:
> I think creating this mixin does not make much sense as it just has same logic inside

I suggest you create `mixin` for usage for whole project for now it's for `color` on other case for `background` with animation. 
You should keep in mind to avoid hardcoded value as you mentioned before
Should change it. Follow rules of our design system
Avoid hardcoded value

Reply:
And what about using multiple transition values?

Reply:
> And what about using multiple transition values?

I think optional third argument using for `mixin` with some default value will be enough

Reply:
> > And what about using multiple transition values?
> 
> I think optional third argument using for `mixin` with some default value in your case and for others will be enough

Reply:
Added mixin for transition and used it. Also added similar mixin for transforms